jfxui/src/main/resources/org/itsallcode/whiterabbit/jfxui/table/days/style.css

```diff
@@ -43,3 +43,11 @@ TableView#day-table {
 .table-row-cell:weekend:selected {
 	-fx-background-color: -fx-focus-color, -fx-cell-focus-inner-border, -fx-selection-bar;
 }
+
+.scroll-bar:vertical {
+    -fx-pref-width: 20;
+}
+
+.scroll-bar:horizontal {
+    -fx-pref-height: 20;
+}
```

Comment:
This affects only the main day table, not activites and reports. If this is your intention, you could move these rules to `rootSceneStyle.css`.

Reply:
Yes, please.

Reply:
done.